build/Dockerfile

```diff
@@ -15,21 +27,22 @@ RUN git clone https://github.com/openshift/ovn-kubernetes/
 WORKDIR ovn-kubernetes/go-controller/
 RUN make windows
 
+# Build Kubernetes node binaries
+WORKDIR /build/
+RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/kubernetes
+WORKDIR /build/kubernetes
+ENV KUBE_BUILD_PLATFORMS windows/amd64
+RUN make WHAT=cmd/kubelet
+RUN make WHAT=cmd/kube-proxy
+
 # download stage for downloading packages
 FROM registry.access.redhat.com/ubi8/ubi-minimal as download
 LABEL stage=download
+RUN mkdir /download/
 WORKDIR /download/
 RUN microdnf -y install wget tar gzip
 RUN microdnf -y update
 
-# Download, checksum and extract the kubernetes node package
-# We are tightly coupling the operator to the OpenShift version, so with every OpenShift release, we will update the
-# kubernetes node version.
-RUN wget https://dl.k8s.io/v1.19.0-rc.2/kubernetes-node-windows-amd64.tar.gz
-RUN echo "61389f8c05c682102e3432a2f05f41b11d531124f61443429627f94ef6e970d44240d44d32aa467b814de0b54a17208b2d2696602ba5dd6d30f64db964900230 kubernetes-node-windows-amd64.tar.gz" > kubernetes-node-windows-amd64.tar.gz.sha512
-RUN sha512sum -c kubernetes-node-windows-amd64.tar.gz.sha512
-RUN tar -zxf kubernetes-node-windows-amd64.tar.gz
-
 # Download, checksum and extract the CNI plugin package
 RUN wget https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-windows-amd64-v0.8.6.tgz
 RUN wget https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-windows-amd64-v0.8.6.tgz.sha512
```

Comment:
We need to update this to having the built from downstream also. Please add a new story to do this or I am fine if you do it here as long as it is reflect in the commit and PR.

Reply:
I will add a new story